src/templates/en/2019/contributors.html

```diff
@@ -315,6 +331,15 @@ <h1 class="title title-lg">
     </div>
 
     <div class="contributor-grid">
+      <a href="https://forms.gle/Qyf3q5pKgdH1cBhq5" class="contributor contributor-join analysts authors brainstormers designers developers editors reviewers translators">
```

Comment:
By listing every team name on this card it will be included in `getFilteredContributorCount` and we'd be showing N+1 contributors per team.

One way around this is to remove the team names and `contributor` class name from this element and add the `.contributor-join` selector to the `.contributor` style rule on line 243.

```suggestion
      <a href="https://forms.gle/Qyf3q5pKgdH1cBhq5" class="contributor-join">
```

```
  .contributor,
  .contributor-join {
    flex: 1;
    ...
}
```

Reply:
I made this change too, thanks for the heads up! I didn't notice the n+1 count before.